### DISCUSSION TITLE
Go implementation of faster-whisper?

### DISCUSSION THREAD

**Discussion by jerichosiahaya** [SYSTRAN/faster-whisper#414] [General]
Hey!

I've been exploring faster-whisper and I'm curious to know if there's a Go version or equivalent available. Has anyone come across a Go implementation or, perhaps, has any experience in adapting the existing one to Go? I believe the Go community could greatly benefit from such a tool given its efficiency and speed.

**Comment by toanhuynhnguyen** [comment]
I think you should use Python for background work and go for the APIs.
I currently implement like this but I use NodeJS for the APIs.
